source/js/apiHelpers.js

```diff
@@ -135,31 +135,31 @@ async function getRecipesByType(type, num) {
 // not sure how to do this
 
 
-/**
```

Comment:
We need Robert to look at this change, as I have no idea how two versions of the same function snuck its way into the main branch and didn't give him any errors on his machine.

Reply:
Resolved in #102